In [ ]:
import sys
print(sys.version)


#### run in Python shell or python script if doesnt work

In [7]:
import cv2
input = cv2.imread('./MasterOpenCV/images/input.jpg')
cv2.imshow('Hello world', input)
cv2.waitKey(0)   # keep open until specified ms, if 0 until keypress
cv2.destroyAllWindows()

#### RESULT: Displays window with image

### Image are stored as numpy

In [4]:
import cv2
import numpy as np

input = cv2.imread('./MasterOpenCV/images/input.jpg')
print input.shape

(830, 1245, 3)


2D dimensions are 830 pixels high and 1245 pixels wide, 3 is the RGB value

In [5]:
print('Height of Image: ', int(input.shape[0]), 'pixels')
print('Weight of Image: ', int(input.shape[1]), 'pixels')

('Height of Image: ', 830, 'pixels')
('Weight of Image: ', 1245, 'pixels')


### Saving images we edit using imwrite()

In [6]:
cv2.imwrite('output.jpg', input)
cv2.imwrite('output.png', input)

True